test/AuthBridgeExecutor.t.sol

```diff
@@ -0,0 +1,665 @@
+        action.values[0]  = value;
+        _queueAction(action);
+    }
+
+    function _encodeHash(Action memory action, uint256 index, uint256 executionTime) internal pure returns (bytes32) {
+        return keccak256(abi.encode(
+            action.targets[index],
+            action.values[index],
+            action.signatures[index],
+            action.calldatas[index],
+            executionTime,
+            action.withDelegatecalls[index]
+        ));
+    }
+
+    function _encodeHash(Action memory action, uint256 executionTime) internal pure returns (bytes32) {
+        return _encodeHash(action, 0, executionTime);
+    }
+
+    function _assertActionSet(uint256 id, bool executed, bool canceled, uint256 executionTime, Action memory actions) internal view {
+        IExecutorBase.ActionsSet memory actionsSet = executor.getActionsSetById(id);
+        assertEq(actionsSet.targets,           actions.targets);
+        assertEq(actionsSet.values,            actions.values);
+        assertEq(actionsSet.signatures,        actions.signatures);
+        assertEq(actionsSet.calldatas,         actions.calldatas);
+        assertEq(actionsSet.withDelegatecalls, actions.withDelegatecalls);
+        assertEq(actionsSet.executionTime,     executionTime);
+        assertEq(actionsSet.executed,          executed);
+        assertEq(actionsSet.canceled,          canceled);
+    }
+
+}
+
+contract AuthBridgeExecutorConstructorTests is AuthBridgeExecutorTestBase {
+
+    function test_constructor_invalidInitParams_boundary() public {
+        vm.expectRevert(abi.encodeWithSignature("InvalidInitParams()"));
+        executor = new AuthBridgeExecutor({
+            delay:        DELAY,
+            gracePeriod:  10 minutes - 1,
+            minimumDelay: 0,
+            maximumDelay: 365 days,
+            guardian:     guardian
+        });
+
+        executor = new AuthBridgeExecutor({
+            delay:        DELAY,
+            gracePeriod:  10 minutes,
+            minimumDelay: 0,
+            maximumDelay: 365 days,
+            guardian:     guardian
+        });
+    }
+
+    function test_constructor() public {
+        vm.expectEmit();
+        emit DelayUpdate(0, DELAY);
+        vm.expectEmit();
+        emit GracePeriodUpdate(0, GRACE_PERIOD);
+        vm.expectEmit();
+        emit GuardianUpdate(address(0), guardian);
+        executor = new AuthBridgeExecutor({
+            delay:        DELAY,
+            gracePeriod:  GRACE_PERIOD,
+            minimumDelay: 0,
+            maximumDelay: 365 days,
+            guardian:     guardian
+        });
+
+        assertEq(executor.getDelay(),       DELAY);
+        assertEq(executor.getGracePeriod(), GRACE_PERIOD);
+        assertEq(executor.getGuardian(),    guardian);
+
+        assertEq(executor.hasRole(executor.DEFAULT_ADMIN_ROLE(), address(this)), true);
+        assertEq(executor.getRoleAdmin(executor.AUTHORIZED_BRIDGE_ROLE()),       executor.DEFAULT_ADMIN_ROLE());
```

Comment:
Alignment

Reply:
This is already aligned. Wdym?

Reply:
Ah, my mistake. Looks weird though that the 2nd arg in a 2nd line is aligned to 3rd arg in the 1st line.